### PR TITLE
docs: AttributeDirectives guide for CLI

### PR DESCRIPTION
**Pull request by wardbell**
See #19510 "CLI prose changes"

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] angular.io application / infrastructure changes
[ ] Other... Please describe:
```

### CODE REVIEW

File: aio/content/guide/attribute-directives.md [hunk 1]
```diff
@@ -43,106 +43,84 @@ when the user hovers over that element. You can apply it like this:
...
-named <code>attribute-directives</code>.
+Create the directive class file in a terminal window with this CLI command.
 
-Create the following source file in the indicated folder:
+<code-example language="sh" class="code-shell">
+ng generate directive highlight
+</code-example>
 
-<code-example path="attribute-directives/src/app/highlight.directive.1.ts" title="src/app/highlight.directive.ts"></code-example>
+The CLI creates `src/app/highlight.directive.ts`, a corresponding test file (`.../spec.ts`, and _declares_ the directive class in the root `AppModule`.
```
Comment (Foxandxss) [inline aio/content/guide/attribute-directives.md]: I find the `.../spec.ts` thing a bit confusing.

File: aio/content/guide/attribute-directives.md [hunk 2]
```diff
@@ -43,106 +43,84 @@ when the user hovers over that element. You can apply it like this:
...
-into the constructor.
-`ElementRef` is a service that grants direct access to the DOM element
-through its `nativeElement` property.
+called `HighlightDirective`, which contains the (currently empty) logic for the directive.
+Exporting `HighlightDirective` makes the directive accessible.
 
-{@a apply-directive}
-
-## Apply the attribute directive
+Now edit the generated `src/app/highlight.directive.ts` to look as follows:
```
Comment (Foxandxss) [inline aio/content/guide/attribute-directives.md]: Maybe it is the result of skimming this PR, but I grow tired of so many "generated" when you specify that file.

**Review by Foxandxss — APPROVED** [review]

File: aio/content/guide/attribute-directives.md [hunk 3]
```diff
@@ -43,106 +43,84 @@ when the user hovers over that element. You can apply it like this:
...
 
-To use the new `HighlightDirective`, create a template that
-applies the directive as an attribute to a paragraph (`<p>`) element.
-In Angular terms, the `<p>` element is the attribute **host**.
+<code-example path="attribute-directives/src/app/highlight.directive.1.ts" title="src/app/highlight.directive.ts"></code-example>
 
-Put the template in its own <code>app.component.html</code>
-file that looks like this:
+The `import` statement specifies an additional `ElementRef` symbol from the Angular `core` library:
```
Comment (gkalpak) [inline aio/content/guide/attribute-directives.md]: Is the `:` intentional?

Comment (Foxandxss) [reply aio/content/guide/attribute-directives.md]: I don't think so

**Comment by vicb** [comment]
missing:
- green CI

**Comment by gkalpak** [comment]
CircleCI didn't find the config and runs its default commands. I tried restarting it a couple of times to no avail 😕

**Comment by jasonaden** [comment]
@wardbell There's one outstanding comment. Also, please rebase and re-push which should fix the CI error. If it doesn't, I'll rebase locally and merge.

**Comment by Foxandxss** [comment]
Sorry @jasonaden , long angular connect week. I will see what can I do.

**Pull request closed without merging**